content_schemas/formats/shared/definitions/_organisation_logo.jsonnet

```
@@ -10,7 +10,6 @@
     crest: {
       type: [
         "string",
-        "null",
       ],
```

```suggestion
      type: "string"
```

minor, you don't need to define this in an array.

---

Thanks.  I'll fix this.